app/models/project.js

```diff
@@ -50,6 +50,15 @@ export default Model.extend({
    */
   customer: belongsTo("customer"),
 
+  /**
+   * Whether the project's comments are visible to the customer
+   *
+   * @property customerVisible
+   * @type {Boolean}
+   * @public
+   */
+  customerVisible: attr("boolean", { defaultValue: false }),
```

Comment:
Since it's a flag, I would prefer `isVisibleToCustomer` over `customerVisible`.

Reply:
That doesn't work. It has to be `customerVisible`.

Reply:
Yeah you're right, it depends on the [backend implementation](https://github.com/adfinis-sygroup/timed-backend/blob/master/timed/projects/models.py#L93)